src/java.desktop/macosx/native/libawt_lwawt/awt/CGraphicsDevice.m

```diff
@@ -58,7 +59,17 @@ static int getBPPFromModeString(CFStringRef mode)
     return 0;
 }
 
-static BOOL isValidDisplayMode(CGDisplayModeRef mode){
+static BOOL isValidDisplayMode(CGDisplayModeRef mode) {
+    // Workaround for apple bug FB13261205, since it only affects arm based macs
+    // and arm support started with macOS 11 ignore the workaround for previous versions
+    if (@available(macOS 11, *)) {
```

Comment:
FYI there's a PR out to make macos 11 the minimum for JDK 22
https://github.com/openjdk/jdk/pull/16155, but this check would be needed in any backport.